release/kata-deploy-binaries.sh

```diff
@@ -127,22 +133,21 @@ install_experimental_kernel() {
 	info "install experimental kernel"
 	DESTDIR="${destdir}" PREFIX="${prefix}" ./kernel/build-kernel.sh -e install
 	popd
+	pushd ${destdir}
```

Comment:
it is any problem if destdir was not cleaned up before?  like call the functions every time in the same machine and and the last created tarballs will include more content, maybe not an fatal error but a bug that will be strange in the future.